packages/api/src/controllers/appeal-search.js

```diff
@@ -0,0 +1,25 @@
+/* istanbul ignore file */
+const { searchAppeal } = require('../lib/db-wrapper');
+const logger = require('../lib/logger');
+const ApiError = require('../lib/api-error');
+
+const appealSearch = async (req, res) => {
+  try {
+    const { searchString } = req.params;
+
+    console.log(searchString);
```

Comment:
So we don't get this logging statement in the prod logs, I think it would be better to either remove the `console.log()` or change it to a debug statement, for example `logger.debug({ seatchString} 'Search string');` so we'd only get it in a dev env.